dashboard/app/helpers/plc/learning_modules_helper.rb

```diff
@@ -2,4 +2,11 @@ module Plc::LearningModulesHelper
   def options_for_plc_task_learning_modules
     Plc::LearningModule.all.pluck(:name, :id).sort
   end
+
+  def options_for_evaluation_answer_modules(course_unit)
+    course_unit.plc_learning_modules.order(:required, :name).map do |learning_module|
```

Comment:
Super small nit: you can `.pluck(:id, :required, :name)` to avoid loading each `LearningModule`.